packages/flutter/lib/src/material/slider.dart

```diff
@@ -578,7 +578,10 @@ class _RenderSlider extends RenderBox {
   void _updateLabelPainter() {
     if (label != null) {
       _labelPainter
-        ..text = new TextSpan(style: _theme.accentTextTheme.body2, text: label)
+        ..text = new TextSpan(
+          style: _sliderTheme.valueIndicatorTextStyle ?? _theme.accentTextTheme.body2,
```

Comment:
seems like we should merge these rather than deciding on one vs the other. that way you can just say "make it bold" (of the valueIndicatorTextStyle) and it'll still be the right font based on whether you're in japanese or chinese or whatever.

Reply:
OK.  I've moved this default to the ThemeData now, but I'll do the merge there.